NEWS.md

```diff
@@ -1,3 +1,6 @@
+# deltaccd 0.2.4
+* Replaced `globalVariable` call to get around `devtools::check` warning with null variable assignment and using the rlang .data variable for ggplot2.
```

Comment:
- function is called `globalVariables`.
- refer to R CMD check instead of just `devtools::check`.

Reply:
Fixed!